src/splunk_otel/env.py

```diff
@@ -14,20 +14,33 @@
 import logging
 import os
 
+from opentelemetry.environment_variables import OTEL_LOGS_EXPORTER, OTEL_METRICS_EXPORTER, OTEL_TRACES_EXPORTER
+from opentelemetry.sdk.environment_variables import (
+    OTEL_ATTRIBUTE_COUNT_LIMIT,
+    OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT,
+    OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT,
+    OTEL_EXPERIMENTAL_RESOURCE_DETECTORS,
+    OTEL_LINK_ATTRIBUTE_COUNT_LIMIT,
+    OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT,
+    OTEL_SPAN_EVENT_COUNT_LIMIT,
+    OTEL_SPAN_LINK_COUNT_LIMIT,
+    OTEL_TRACES_SAMPLER,
+)
+
 DEFAULTS = {
-    "OTEL_PYTHON_LOG_CORRELATION": "true",  # FIXME: revisit
-    "OTEL_TRACES_EXPORTER": "otlp",
-    "OTEL_METRICS_EXPORTER": "otlp",
-    "OTEL_LOGS_EXPORTER": "otlp",
-    "OTEL_ATTRIBUTE_COUNT_LIMIT": "",
-    "OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT": "",
-    "OTEL_SPAN_EVENT_COUNT_LIMIT": "",
-    "OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT": "",
-    "OTEL_LINK_ATTRIBUTE_COUNT_LIMIT": "",
-    "OTEL_SPAN_LINK_COUNT_LIMIT": "1000",
-    "OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT": "12000",
-    "OTEL_EXPERIMENTAL_RESOURCE_DETECTORS": "host,process",
-    "OTEL_TRACES_SAMPLER": "always_on",
+    "OTEL_PYTHON_LOG_CORRELATION": "true",  # used by opentelemetry-instrumentation-logging (installed by bootstrap)
```

Comment:
My understanding is that 
```
OTEL_PYTHON_LOG_CORRELATION=true
```
is "destructive" (modifies the behavior of stdout/file logging in a way that is noticeable to the app's environment), while I think what we want is
```
OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED=true
```
which sends the logs (with trace context) through otlp by default and does nothing to the configured logging at the python level.
This is from old notes I made on this topic and may be incorrect.  Please research and consult with me and product management.

Reply:
Thanks. Discussed with product and fixed.